ql/src/test/queries/clientpositive/udf_array_distinct.q

```diff
@@ -0,0 +1,33 @@
+--! qt:dataset:src
+set hive.fetch.task.conversion=more;
```

Comment:
There are too many select *,
better to add
``-- SORT_QUERY_RESULTS`` to avoid any flakiness

Reply:
Added flag